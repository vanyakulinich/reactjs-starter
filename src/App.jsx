import React from "react"

const App = () => {
  // TODO: entry point: all providers should be here

  return <>Starter</>
}

export default App
