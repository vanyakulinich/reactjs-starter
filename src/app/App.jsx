const App = () => {
  // TODO: entry point: all providers should be here

  return (
    <>
      TEST
      <div>
        test
        <div>test</div>
      </div>
    </>
  );
};

export default App;
