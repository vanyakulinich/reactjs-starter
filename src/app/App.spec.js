import { render } from "enzyme";
import App from "./App";

describe("App Main Entry Component", () => {
  it("should render App component", () => {
    const app = render(<App />);
    expect(app).toMatchSnapshot();
  });
});
