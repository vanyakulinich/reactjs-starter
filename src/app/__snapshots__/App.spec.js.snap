// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Main Entry Component should render App component 1`] = `
initialize {
  "0": Node {
    "data": "TEST",
    "next": Node {
      "attribs": Object {},
      "children": Array [
        Node {
          "data": "test",
          "next": Node {
            "attribs": Object {},
            "children": Array [
              Node {
                "data": "test",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": [Circular],
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          "parent": [Circular],
          "prev": null,
          "type": "text",
        },
        Node {
          "attribs": Object {},
          "children": Array [
            Node {
              "data": "test",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": Node {
            "data": "test",
            "next": [Circular],
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
      ],
      "name": "div",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": Node {
        "children": Array [
          [Circular],
          [Circular],
        ],
        "name": "root",
        "next": null,
        "parent": null,
        "prev": null,
        "type": "root",
      },
      "prev": [Circular],
      "type": "tag",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "parent": Node {
      "children": Array [
        [Circular],
        Node {
          "attribs": Object {},
          "children": Array [
            Node {
              "data": "test",
              "next": Node {
                "attribs": Object {},
                "children": Array [
                  Node {
                    "data": "test",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
            Node {
              "attribs": Object {},
              "children": Array [
                Node {
                  "data": "test",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": Node {
                "data": "test",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
      ],
      "name": "root",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
    },
    "prev": null,
    "type": "text",
  },
  "1": Node {
    "attribs": Object {},
    "children": Array [
      Node {
        "data": "test",
        "next": Node {
          "attribs": Object {},
          "children": Array [
            Node {
              "data": "test",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "text",
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {},
          "x-attribsPrefix": Object {},
        },
        "parent": [Circular],
        "prev": null,
        "type": "text",
      },
      Node {
        "attribs": Object {},
        "children": Array [
          Node {
            "data": "test",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "text",
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Node {
          "data": "test",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "text",
        },
        "type": "tag",
        "x-attribsNamespace": Object {},
        "x-attribsPrefix": Object {},
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": Node {
      "children": Array [
        Node {
          "data": "TEST",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "text",
        },
        [Circular],
      ],
      "name": "root",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
    },
    "prev": Node {
      "data": "TEST",
      "next": [Circular],
      "parent": Node {
        "children": Array [
          [Circular],
          [Circular],
        ],
        "name": "root",
        "next": null,
        "parent": null,
        "prev": null,
        "type": "root",
      },
      "prev": null,
      "type": "text",
    },
    "type": "tag",
    "x-attribsNamespace": Object {},
    "x-attribsPrefix": Object {},
  },
  "_root": [Circular],
  "length": 2,
  "options": Object {
    "decodeEntities": true,
    "xml": false,
  },
}
`;
